### DISCUSSION TITLE
Multichannel config example

### DISCUSSION THREAD

**Discussion by R1ch1e-Lee** [rtl-airband/RTLSDR-Airband#475] [General]
Working on a new config file to run multichannel mode. I am confusing myself with a config file and streaming 2 channels to broadcastify as a left and right audio stream.   If someone could share an example working multichannel config file I would really appreciate it.
Thanks in advance for any help

**Comment by buentAir** [comment]
I solved it with an MP3 channel, no streaming. However, it might help you get started. I created a mixer in which I then combine several channels. The audio can be controlled to the left or right side. The information on this is stored in the wiki: [Mixers](https://github.com/charlie-foxtrot/RTLSDR-Airband/wiki/Configuring-mixers)
